playbook.yaml

```diff
@@ -1,55 +1,55 @@
 ---
- - hosts: all
-   pre_tasks:
-     - name: Gather facts from ALL hosts (regardless of limit or tags)
-       setup:
-       delegate_to: "{{ item }}"
-       delegate_facts: True
-       when: hostvars[item]['ansible_default_ipv4'] is not defined
-       with_items: "{{ groups['all'] }}"
-     - name: Copy files
-       include_tasks: tasks/copy_files.yaml
-       loop: '{{ ["all"] + group_names + [inventory_hostname_short] }}'
-       loop_control:
-         loop_var: myhost
+- hosts: all
+  pre_tasks:
+    - name: Gather facts from ALL hosts (regardless of limit or tags)
+      setup:
+      delegate_to: "{{ item }}"
+      delegate_facts: True
+      when: hostvars[item]['ansible_default_ipv4'] is not defined
+      with_items: "{{ groups['all'] }}"
+    - name: Copy files
+      include_tasks: tasks/copy_files.yaml
+      loop: '{{ ["all"] + group_names + [inventory_hostname_short] }}'
+      loop_control:
+        loop_var: myhost
```

Comment:
are these changes necessary? If not I would suggest we avoid formatting changes in this PR to avoid noise and do that in a follow-up PR

Reply:
I think that will be great and will be good for posterity, so that we can come back and see what did it took to make it work in 2024 with the latest ubuntu 2204.